dev-clean/benchmarks/external_lib/assert_fix.c

```diff
@@ -0,0 +1,23 @@
+#include <stdbool.h>
+
+void llsc_assert(bool);
+
+int main() {
+  int x, y, z;
+  make_symbolic(&x, sizeof(x));
+  make_symbolic(&y, sizeof(y));
+  make_symbolic(&z, sizeof(z));
+  if (x == 2 && y == 3) {
+    llsc_assert(x + y == 5);
+    // before, the path condition here will contain both x == 2, y == 3, x + y != 5
```

Comment:
maybe just describe what's the expected behavior here; after a few months, we probably will forget the "before"/"after" situation

Reply:
Definitely, Thanks, I write the expected behavior.